receiver/aerospikereceiver/config_test.go

```diff
@@ -18,62 +18,71 @@ import (
 	"testing"
 	"time"
 
+	as "github.com/aerospike/aerospike-client-go/v5"
 	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/config/configtls"
 )
 
 func TestValidate(t *testing.T) {
 	testCases := []struct {
 		name     string
 		config   *Config
 		expected error
-	}{
+	}{ // TODO add tls tests
```

Comment:
We try not to allow TODOs in the codebase. Please resolve or capture as an issue.

Reply:
My bad. This is out of date, removed it.